### ISSUE TITLE
NOK: Minting limit exceed on polygon Retrying #120

### ISSUE THREAD

**Issue by harmen07** [codeSTACKr/create-10k-nft-collection#100]
### Version

- [X] Yes

### General Question

- [X] No

### What happened?

As mentioned in the comments, Subscription limit is what causes the error "NOK: Minting limit exceed on polygon Retrying". 

Mint limit was not addressed in the tutorials. do we have to restart the entire process after getting a subscription, or we can just start back from where we left off ?

### What have you tried?

I Googled.

### Relevant log output

```shell
NOK: Minting limit exceed on polygon Retrying
```

### Protect your information

- [X] I Agree

**Comment by anthonyiu** [comment]
Same issue here...

**Comment by cserra92** [comment]
same here! I'm trying to find a workaround without having to re-do everything since I already listed what was able to mint via OpenSea

**Comment by cserra92** [comment]
yeah $100 a month seems excessive to only be able to mint 5k. also kind of misleading to say in the videos 10k for free when it’s clearly not, but it was definitely useful for a learning experience at the very least

**Comment by JeRazMeta** [comment]
Same issue here. Now, NFTport allows only 100 nft to mint free. I'm still looking for a solution or an alternative way to mint.

**Comment by Markkevin052** [comment]
but in my case from start its not minting 
not even 1
